### PR TITLE
Jdp200202 16

### PR DESCRIPTION
**Pull request by EmiCoder**

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+@NoArgsConstructor
+@Entity
+@Table(name="users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+
+    @Column(name="firstName")
```
Comment (g-chmura) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]:
```suggestion
    @Column(name="first_name")
```

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private String firstName;
+
+    @Column(name="lastName")
```
Comment (g-chmura) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]:
```suggestion
    @Column(name="last_name")
```

File: src/main/java/com/kodilla/ecommercee/dto/ProductDto.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+}
```
Comment (g-chmura) [inline src/main/java/com/kodilla/ecommercee/dto/ProductDto.java]: Na ten moment Dto mają być puste

File: src/main/java/com/kodilla/ecommercee/domain/User.java (hunk 1 above)
Comment (mateusz58) [reply src/main/java/com/kodilla/ecommercee/domain/User.java]: Column name in database should consist only of small letters and _  playing role of separator between words

**Pull request merged**